icu4c/source/i18n/number_modifiers.cpp

```diff
@@ -60,6 +60,41 @@ Modifier::Parameters::Parameters(
     const ModifierStore* _obj, Signum _signum, StandardPlural::Form _plural)
         : obj(_obj), signum(_signum), plural(_plural) {}
 
+bool Modifier::semanticallyEquivalent(const Modifier& other) const {
+    Parameters params0;
+    Parameters params1;
```

Comment:
suggestion: this function be more readable with more specific names of these variables, for example thisParams vs. otherParams

Reply:
+1

Reply:
There is no "this" and "other" because this is a static function. There is A and B, Left and Right, or 0 and 1. I chose 0 and 1.

Reply:
Clarification: in Java it is a static function. In C++ it is a member function.

Reply:
Okay now that this is no longer a static function, I changed to This/Other in both C++ and Java